skyline-vscode/src/skyline_session.ts

```diff
@@ -6,6 +6,8 @@ const fs = require('fs');
 
 import {Socket} from 'net';
 import { simpleDecoration } from './decorations';
+import { energy_component_type_mapping } from './utils';
+import { privateEncrypt } from 'crypto';
```

Comment:
Is this module used anywhere?

Reply:
I am only using energy_component_type_mapping , I don't see privateEncrypt  being used in that file